docs/proxy/atlas/metadata_configs.md

```diff
@@ -0,0 +1,210 @@
+                "relationshipCategory": "COMPOSITION",
+                "relationshipLabel": "__Reader.Users",
+                "endDef1": {
+                    "type": "Reader",
+                    "name": "user",
+                    "isContainer": False,
+                    "cardinality": "SINGLE",
+                    "isLegacyAttribute": True
+                },
+                "endDef2": {
+                    "type": "User",
+                    "name": "entityReads",
+                    "isContainer": True,
+                    "cardinality": "SET",
+                    "isLegacyAttribute": True
+                },
+                "propagateTags": "NONE"
+            },
+            {
+                "name": "Metadata_Reader",
+                "typeVersion": "2.0",
+                "relationshipCategory": "COMPOSITION",
+                "relationshipLabel": "__Metadata.Reader",
+                "endDef1": {
+                    "type": "Metadata",
+                    "name": "readers",
+                    "isContainer": True,
+                    "cardinality": "SET",
+                    "isLegacyAttribute": False
+                },
+                "endDef2": {
+                    "type": "Reader",
+                    "name": "entityMetadata",
+                    "isContainer": False,
+                    "cardinality": "SINGLE",
+                    "isLegacyAttribute": True
+                },
+                "propagateTags": "NONE"
+            },
+            {
+                "name": "DataSet_Metadata",
+                "typeVersion": "2.0",
+                "relationshipCategory": "AGGREGATION",
+                "relationshipLabel": "_DataSet.Metadata",
+                "endDef1": {
+                    "type": "DataSet",
+                    "name": "metadata",
+                    "isContainer": True,
+                    "cardinality": "SINGLE",
+                    "isLegacyAttribute": True
+                },
+                "endDef2": {
+                    "type": "Metadata",
+                    "name": "parentEntity",
+                    "isContainer": False,
+                    "cardinality": "SINGLE",
+                    "isLegacyAttribute": True
+                },
+                "propagateTags": "NONE"
+            },
+        ]
+    }
+    try:
+        client.typedefs.create(data=userReaderMetadata_DefDict)
+    except Conflict as ex:
+        print("Exception: {0}".format(str(ex)))
+        client.typedefs.update(data=userReaderMetadata_DefDict)
+```
+
+Popular Tables
+--------------
+Amundsen has a concept of popular tables, which is a default entry point of the application for now. 
+Popular Tables API expects you to generate the relationships above, and fill in your `popularityScore` within Metadata entityt
+for each Table. 
+The suggested formula to generate the popularity score is
+```
+Popularity score = number of distinct readers * log(total number of reads)
```

Comment:
Do you guys also parse some query access log to get the table distinct readers?

Reply:
Yes. We parse Apache Ranger's access log to get this done. Ranger has the ability to save the logs in Solr and Hive both, and we decided to go with hive to get/parse the access logs. One of our airflow dag parses the logs and creates entities in atlas. (the sample code snippet is also attaches in the documentation)